docs/integration/windows-integration.md

```diff
@@ -0,0 +1,75 @@
+title: Sematext Windows Monitoring
+description: Monitor Windows bare metal servers and VMs using a lightweight agent and visualize your full infrastructure in Sematext Cloud
+
+
+## Windows Services
+
+Sematext Monitoring now includes support for Windows Services. Starting from Sematext Agent [version 3.5.0](https://sematext.com/docs/agents/sematext-agent/releasenotes/#version-350), you can collect and monitor the status of any Windows Service available in your system.
+
+### Metrics
+
+The [Sematext Agent for Windows](https://sematext.com/docs/agents/sematext-agent/windows-installation/), starting from version 3.5.0, is capable of collecting metrics related with the status of Windows services. At present, the primary metric supported is the status of each service including a number of statistics, which allows users to determine whether a service is running, stopped, or experiencing issues.
+
+
+#### List of available metrics
+
+|Name| Type  | Description |
+|----|-------|-------------|
+|windows.service.status            | number |status of a Windows service|
+|windows.service.total             | gauge  |total number of Windows services|
```

Comment:
@cpipilas repeating the old comment about it being a little weird to have this metric in this table because other metrics are pertaining to a single WS instance, while this is a count of all WSs.

Reply:
I can just change it once this is merged/published.